src/app_service/service/accounts/service.nim

```diff
@@ -22,6 +22,10 @@ logScope:
 const PATHS = @[PATH_WALLET_ROOT, PATH_EIP_1581, PATH_WHISPER, PATH_DEFAULT_WALLET]
 const ACCOUNT_ALREADY_EXISTS_ERROR =  "account already exists"
 
+# allow runtime override via environment variable. core contributors can set a
+# specific peer to set for testing messaging and mailserver functionality with squish.
+let TEST_PEER_ENR = getEnv("TEST_PEER_ENR").string
```

Comment:
Maybe you can delete that one if you don't use it: https://github.com/status-im/status-desktop/pull/6946/files

also nit: `$getEnv("TEST_PEER_ENR")`